apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-create-edit.component.spec.ts

```diff
@@ -426,7 +426,12 @@ describe('DotTemplateCreateEditComponent', () => {
             beforeEach(() => {
                 const storeMock = jasmine.createSpyObj(
                     'DotTemplateStore',
-                    ['saveTemplate', 'goToTemplateList', 'goToEditTemplate'],
+                    [
+                        'saveTemplate',
+                        'saveAndPublishTemplate',
+                        'goToTemplateList',
+                        'goToEditTemplate'
+                    ],
```

Comment:
`jasmine.createSpyObj()` should receive two values and we are sending three here. We can `mock` the values of the DotTemplateStore provider like this: 

```---js
const storeMock = {
      saveTemplate: jasmine.createSpy(),
      saveAndPublishTemplate: jasmine.createSpy(),
      goToTemplateList: jasmine.createSpy(),
      goToEditTemplate: jasmine.createSpy(),
      vm$: of({
                original: {
                    ...EMPTY_TEMPLATE_DESIGN,
                    identifier: '123',
                    title: 'Some template'
                },
                apiLink: '/api/link'
       })
};
```